netutils/config/parser.py

```diff
@@ -49,49 +60,54 @@ def __init__(self, config):
         super(BaseSpaceConfigParser, self).__init__(config)
 
     @property
-    def indent_level(self):
+    def banner_end(self) -> str:
```

Comment:
How is this diff then what it was inherited from?

Reply:
Sorry I don't understand - do you mean why the signature of this function is different than what it is inheriting from? If so, I don't know which exact function you're referring to because to me it seems like the signatures match here and also Mypy [should catch this](https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides)